### PR TITLE
refactored comments added all html pages

### PR DESCRIPTION
**Pull request by varnita619**

### CODE REVIEW

**Comment by SJTGSHIVAM** [comment]
Refactoring looks fine. These changes are good. Some suggestions apart from these:
1.  Either make the navbar opaque or hide it on scroll.
2.  you can change images some of them look stretched.

**Review by SJTGSHIVAM — APPROVED** [review]

File: src/Pages/Home/HomePage.jsx [hunk 1]
```diff
@@ -6,88 +6,86 @@ import { Link } from 'react-router-dom';
...
+                    <div className="horizontal-card-image">
+                        <img src="https://i.pinimg.com/originals/ba/97/c1/ba97c196f9aefc9513c9bc05039cb336.jpg" alt="card-image" />
+                    </div>
+                    <div className="card-content">
+                        <div className="card-details">
+                            <h3>Summer Arrivals</h3>
+                            <span>New collection</span>
+                        </div>
+                        <div className="brand-image-container">
+                            <img src='https://pngimg.com/uploads/gucci/gucci_PNG22.png' style={{ width: '6.5rem', height: '2.5rem' }} className='brand-image'></img>
```
Comment (anuj-kumary) [inline src/Pages/Home/HomePage.jsx:53]: Try to not use inline styling

**Review by anuj-kumary — APPROVED** [review]
- Clean code
- Good naming conventions
- Good file structure

File: src/Pages/Home/HomePage.jsx [hunk 2]
```diff
@@ -6,88 +6,86 @@ import { Link } from 'react-router-dom';
...
-                    <span>New collection</span>
-                </div>
-                <div className="brand-image-container">
-                <img src='https://upload.wikimedia.org/wikipedia/commons/thumb/5/53/H%26M-Logo.svg/1200px-H%26M-Logo.svg.png' style={{width:'6.5rem', height:'2.5rem'}} className='brand-image'></img>
-                
-                </div>
-                <span className='discount-details'>Upto 30% Off</span>
+            <div className="banner">
+                <img src="https://res.cloudinary.com/dqgqdj4jf/image/upload/v1647587862/banner/home_xci4gu.gif"
+                    alt="" />
```
Comment (Rohanmond) [inline src/Pages/Home/HomePage.jsx:11]: add some alt

File: src/Pages/Home/HomePage.jsx [hunk 3]
```diff
@@ -6,88 +6,86 @@ import { Link } from 'react-router-dom';
...
+                    <div className="horizontal-card-image">
+                        <img src="https://images.unsplash.com/photo-1544610280-5265bdceba0c?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8M3x8bGV2aXN8ZW58MHx8MHx8&w=1000&q=80" alt="card-image" />
+                    </div>
+                    <div className="card-content">
+                        <div className="card-details">
+                            <h3>Summer Arrivals</h3>
+                            <span>New collection</span>
+                        </div>
+                        <div className="brand-image-container">
+                            <img src='https://upload.wikimedia.org/wikipedia/commons/thumb/7/75/Levi%27s_logo.svg/2560px-Levi%27s_logo.svg.png' style={{ width: '6.5rem', height: '2.5rem' }} className='brand-image'></img>
```
Comment (Rohanmond) [inline src/Pages/Home/HomePage.jsx:76]: try not to use inline styling

**Review by Rohanmond — APPROVED** [review]

**Review by mdarif6 — APPROVED** [review]
-check header while page scrolling, either fix it or need to move upward along with page content.
-as in other comments, try to keep styling in different file.
- I also doing same mistake in alt="". so give some vale to it.

**Pull request merged**